tools/transformer/src/name.ignore.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This is only temporary. We have CLOUDP-165932 to remove those models from ignore list.